src/data/[provider]-spark-retrieval-timings-summary.json.js

```diff
@@ -9,9 +9,10 @@ const {
     options: { provider: { type: "string" } }
 });
 
-const start = '2024-04-07';
+const start = '2025-01-14';
 const end = getDateXDaysAgo(1);
 
 const summary = await pRetry(() => jsonFetcher(`https://stats.filspark.com/miner/${provider}/retrieval-timings/summary?from=${start}&to=${end}`), { retries: 3 });
 
-process.stdout.write(JSON.stringify(summary));
+// fake datapoint on January 13th so chart starts from 0
+process.stdout.write(JSON.stringify([{ day: '2025-01-13', ttfb_ms: 0 }, ...summary]));
```

Comment:
I prefer to configure the chart to start from zero instead of adding fake data points.

See https://observablehq.com/plot/marks/axis and https://github.com/observablehq/plot/discussions/1938

> use the `y: {zero: true}` option, or add a mark that includes `0` as a value for y (typically, `Plot.ruleY([0])`).

Reply:
The fake data point also creates a spike in the chart, creating the false impression that TTFB was 0ms on Jan 13th.

<img width="703" alt="Screenshot 2025-01-16 at 17 11 29" src="https://github.com/user-attachments/assets/decf8c72-b8b2-45bf-88c7-43fb15159052" />

If it's too much work to configure the Y axis to start from `0`, then I am fine with reverting back to your original solution rather than adding hacks.

Reply:
Ahh I see what you mean, there was a slight misunderstanding on my part. I've removed the fake datapoints.